openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -1421,9 +1422,161 @@ def get_render_products(self):
                 colorspace=lib.get_color_management_output_transform()
             )
             products.append(product)
+        return products
+
+
+class RenderProducts3Delight(ARenderProducts):
+    """Expected files for 3Delight renderer.
+
+    Warning:
+        This is very rudimentary and needs more love and testing.
+    """
+
+    renderer = "_3delight"
+
+    def __init__(self, layer, render_instance):
+        super(RenderProducts3Delight, self).__init__(layer, render_instance)
+
```

Comment:
```suggestion
    def get_multipart(self):
        return False
  
```
I think you need to include this as the collector of render layer errors out if they find there is no get_multipart args

Reply:
This seems resolved?

Reply:
yes it is